docs/source/guides/export_model.mdx

```diff
@@ -226,6 +231,7 @@ optimum-cli export neuron --model stabilityai/stable-diffusion-2-1-base \
   --batch_size 1 \
   --height 512 `# height in pixels of generated image, eg. 512, 768` \
   --width 512 `# width in pixels of generated image, eg. 512, 768` \
+  --num_image_per_prompt 4 `# number of images to generate per prompt, defaults to 1` \
```

Comment:
is this needed with dynamic batching support now?

Reply:
If we turn on dynamic batching, we can input any batch size / number images per prompt, but by default the dynamic batch size is turned off, so we can keep the snippet this way I think.